tests/ast/data/col_in_.test

```diff
@@ -0,0 +1,19 @@
+## TEST CASE
+
+df = session.table("test_table")
+
+df = df.select(col("NUM").in_(1, 2, 3))
+
+df = df.select(col("NUM").in_(df))
+
+df = df.select(col("NUM").in_([1, 2, 3]))
+
+## EXPECTED OUTPUT
+
+res1 = session.table("test_table")
+
+res2 = res1.select(col("NUM").in_(1, 2, 3))
+
+res3 = res2.select(col("NUM").in_(res2))
```

Comment:
Is it possible to use multiple dataframes here? Or only a single one? If the latter, maybe a quick validation/error would be good to have. If not, let's have a test case.

Reply:
Seems like only one DataFrame is allowed as a parameter. If multiple are used, the existing Snowpark code has parameter validation which raises the following exception, which confirms this as well.
```
TypeError: '<class 'snowflake.snowpark._internal.analyzer.expression.ScalarSubquery'>' is not supported for the values parameter of the function in(). You must either specify a sequence of literals or a DataFrame that represents a subquery.
```